pkg/sdk/sdk.go

```diff
@@ -0,0 +1,147 @@
+package sdk
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/PereRohit/util/log"
+	respModel "github.com/PereRohit/util/model"
+	"github.com/gorilla/mux"
+	"io"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+	"time"
+)
+
+type htmlToPdfSvc struct {
+	svcUrl string
```

Comment:
Instead of creating a http client for every request, create the http client once when `NewHtmlToPdfSvc` is called and assign it in the receiver struct field. Then use this for downstream http calls.

Reply:
done